src/views/UserView.vue

```diff
@@ -0,0 +1,126 @@
+<template>
+  <div align="center">
+    <div class="ppBox">
+      <img
+        src="https://www.w3schools.com/howto/img_avatar2.png"
+        alt="Avatar"
+      >
+    </div>
+    <div class="dataBox">
+      <h2>Account details</h2>
+      Address: {{ address }}<br>
+      Name: {{ user.alias }}<br>
+      Council status: {{ user.councilStatus }}
+      <div
+        v-if="address == $store.getters['common/wallet/address']"
+        style="display: inline"
+      >
+        <button
+          v-if="user.councilStatus == 'unavailable'"
+          @click="register()"
+        >
+          Register for council
+        </button>
+        <button
+          v-if="user.councilStatus == 'available'"
+          @click="deRegister()"
+        >
+          Deregister from council
+        </button>
+      </div>
+      <br>
+      Vote rights: {{ user.voteRights.length }}
+      <button
+        v-if="address == $store.getters['common/wallet/address']"
+        @click="$router.push({name: 'Vote'})"
+      >
+        Vote
+      </button>
+      <br>
+      Owned card schemes: {{ user.ownedCardSchemes.length }} <br>
```

Comment:
we call this card frames now, card scheme is the old name which confused too many test users (In the blockchain we still call it that way, but this might also change over the long run)